src/scripts/core/visualization-container.js

```diff
@@ -74,6 +74,12 @@
             return this.ctx;
         },
 
+        setVisibility: function (visible) {
+            var node = this.layer.node();
+
+            visible ? $(node).show() : $(node).hide();
```

Comment:
The problem is with your assumption that jquery if available on the global scope --- Are you saying every project which uses Contour now needs to also have jquery on the global scope? If yes, that's a dependency (and an un-necessary one). 

For e.g., see the docs on http://forio.com/contour/documentation.html#quickstart for including contour on a project; that (correctly) has no mention of jquery. If someone just followed the directions on that page to include contour, your code will not work.